F1/F1/Race/RaceViewModel.swift

```diff
@@ -9,45 +9,65 @@ import Foundation
 
 class RaceViewModel {
 
+    // MARK: Variables
     private var repository: RaceRepositoryType?
     private weak var delegate: ViewModelDelegate?
     private(set) var race: RaceInfo?
-    private(set) var sortedRaceSessions: [(key: String, value: RaceSessions)] = []
+    private(set) var sortedRaceSession: [RaceSessionDetail] = []
 
     init(repository: RaceRepositoryType,
          delegate: ViewModelDelegate) {
```

Comment:
this can be on the same line. Only put parameters on a new line if they go past the breakpoint. It's not a biggie, just the standard we're following for now. SwiftLint should have said that for you